liveblog.php

```diff
@@ -57,6 +57,8 @@
 	public static $is_rest_api_call       	= false;
 	public static $auto_archive_days     	= null;
 	public static $auto_archive_expiry_key  = 'liveblog_autoarchive_expiry_date';
+	
+	public static $supported_post_types = array();
```

Comment:
Why is a static variable better here than a regular class variable?

Reply:
The init method is static. As far as I know PHP allow non-static members to be accessed in a static method.

Reply:
Ahh, you're right - the `is_liveblog_post()` method is static too and it's going to rely on this.